plugins/kotlin/j2k/shared/src/org/jetbrains/kotlin/nj2k/tree/expressions.kt

```diff
@@ -72,6 +72,15 @@ class JKQualifiedExpression(
     override fun accept(visitor: JKVisitor) = visitor.visitQualifiedExpression(this)
 }
 
+class JKArrayGetterExpression(
```

Comment:
Probably `JKArrayAccessExpression` is a better name, which is consistent with Java and Kotlin PSI names.